website/layouts/functions.erb

```diff
@@ -5,14 +5,14 @@
           <li<%= sidebar_current("docs-config-index") %>>
             <a class="back" href="/docs/configuration/index.html">Terraform Language</a>
           </li>
-  
+
           <li<%= sidebar_current("docs-config-functions") %>>
             <a class="back" href="/docs/configuration/functions.html">Functions</a>
           </li>
-  
+
           <li<%= sidebar_current("docs-funcs-numeric") %>>
-            <a href="#">Numeric Functions</a>
-            <ul class="nav nav-visible">
+            <a href="#docs-funcs-numeric">Numeric Functions</a>
+            <ul class="nav nav-visible" id="docs-funcs-numeric">
```

Comment:
(this is just to fix an annoying behavior where it would jump to the top of the page and lose your spot if you clicked on one of these "header" links in the nav.)